test/uk/gov/hmrc/plasticpackagingtax/registration/views/group/OrganisationDetailsTypeViewSpec.scala

```diff
@@ -137,9 +137,7 @@ class OrganisationDetailsTypeViewSpec extends UnitViewSpec with Matchers {
 
       radioInputMustBe(1, UK_COMPANY)
       radioInputMustBe(2, PARTNERSHIP, Some("organisationDetails.type.GroupPartnership"))
-      radioInputMustBe(3, CHARITABLE_INCORPORATED_ORGANISATION)
```

Comment:
These indexes are not important; this doesn't break the mappings in the controller.